components/Pledge/Form.js

```diff
@@ -139,160 +161,203 @@ class Pledge extends Component {
     } = this.state
 
     const {
-      loading, error
+      loading, error, isMember, t, me, statement, query, packages
     } = this.props
 
-    return (
-      <Loader loading={loading} error={error} render={() => {
-        const {
-          query, me, t,
-          crowdfunding,
-          receiveError,
-          crowdfundingName
-        } = this.props
+    const queryPackage = query.package && query.package.toUpperCase()
+    const statementTitle = statement && t(`pledge/form/statement/${queryPackage}/title`, statement)
 
-        if (crowdfunding.hasEnded && !this.props.pledge) {
-          return (
-            <div>
-              <H1>{t('pledge/title')}</H1>
-              <RawHtml type={P} dangerouslySetInnerHTML={{
-                __html: t('ended/pledge/lead')
-              }} />
-            </div>
-          )
-        }
+    const meta = statementTitle
+      ? {
+        title: t('pledge/form/statement/share/title', statement),
+        description: t('pledge/form/statement/share/description'),
+        image: `${ASSETS_SERVER_BASE_URL}/render?width=1200&height=628&updatedAt=${encodeURIComponent(statement.updatedAt)}&url=${encodeURIComponent(`${PUBLIC_BASE_URL}/community?share=${statement.id}&pkg=${queryPackage}`)}`
+      }
+      : {
+        title: t('pledge/meta/title'),
+        description: t('pledge/meta/description'),
+        image: `${CDN_FRONTEND_BASE_URL}/static/social-media/logo.png`
+      }
 
-        const showSignIn = this.state.showSignIn && !me
+    return (
+      <Fragment>
+        <Meta data={meta} />
+        <Loader loading={loading} error={error} render={() => {
+          const {
+            receiveError,
+            crowdfundingName,
+            hasEnded
+          } = this.props
 
-        const pkg = query.package
-          ? crowdfunding.packages.find(
-            pkg => pkg.name === query.package.toUpperCase()
-          )
-          : null
-        const userPrice = !!query.userPrice
+          if (hasEnded && !this.props.pledge) {
+            return (
+              <div>
+                <H1>{t('pledge/title')}</H1>
+                <RawHtml type={P} dangerouslySetInnerHTML={{
+                  __html: t('ended/pledge/lead')
+                }} />
+              </div>
+            )
+          }
 
-        return (
-          <div>
-            <H1>{t('pledge/title')}</H1>
+          const showSignIn = this.state.showSignIn && !me
 
-            {!!receiveError && (
-              <P style={{ color: colors.error, marginBottom: 40 }}>
-                <RawHtml dangerouslySetInnerHTML={{
-                  __html: receiveError
-                }} />
-              </P>
-            )}
+          const pkg = queryPackage
+            ? packages.find(
+              pkg => pkg.name === queryPackage
+            )
+            : null
+          const userPrice = !!query.userPrice
+
+          return (
```

Comment:
The rendering logic is quite convoluted here: the criteria whether to show the `Accordion` overview, a single form, or the social media preview is hard to reason about.